### PR TITLE
Fix Typo

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
Fixed some typos

### CODE REVIEW

**Comment by satyarohith** [comment]
@Isaiah-Hamilton, run `yarn fmt` and push the changes.

**Comment by Isaiah-Hamilton** [comment]
@satyarohith I ran `yarn fmt` and pushed the changes.

**Comment by satyarohith** [comment]
@Isaiah-Hamilton, you are using an older version of `deno`. Please upgrade to the latest (run `deno upgrade`) and then run `yarn fmt`.

**Comment by Isaiah-Hamilton** [comment]
@satyarohith sorry about that.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "deno_website2",
+  "name": "denoland",
```
Comment (satyarohith) [inline package.json:2]:
```suggestion
  "name": "dotland",
```

**Comment by satyarohith** [comment]
Thanks

**Pull request merged**